clang/include/clang/Lex/ModuleMap.h

```diff
@@ -541,11 +541,14 @@ class ModuleMap {
   ///
   /// \param IsExplicit Whether this is an explicit submodule.
   ///
-  /// \returns The found or newly-created module, along with a boolean value
-  /// that will be true if the module is newly-created.
-  std::pair<Module *, bool> findOrCreateModule(StringRef Name, Module *Parent,
-                                               bool IsFramework,
-                                               bool IsExplicit);
+  /// \returns The found or newly-created module.
+  Module *findOrCreateModule(StringRef Name, Module *Parent, bool IsFramework,
```

Comment:
This breaks the lldb build, which uses the returned boolean value here: https://github.com/llvm/llvm-project/blob/0d0abb351b5fcf49ccc46eba8b7f2a1f353a05a6/lldb/source/Plugins/TypeSystem/Clang/TypeSystemClang.cpp#L1233

Reply:
Thanks for the heads-up, I'll make a forward fix ASAP.

Reply:
Fixed in 19131c7f.